SU-SWS/stanford_basic
# D8CORE-2565: adding margin below the secondary menu

# READY FOR REVIEW

# Summary
- Adding a margin bottom to the secondary menu to keep a long menu from bumping against the footer when the content well is short.

# Review By (Date)
- Monday

# Urgency
- low

# Steps to Test

1. Pull in this change.
2. Create a page with a couple of lines in the body content.
3. Create a secondary menu with many pages. About 5 will do it.
4. Verify that the menu doesn't touch the footer.

# Affected Projects or Products
- SOE, Basic

# Associated Issues and/or People
- D8CORE-2565
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
